packages/database/src/migrations/20200914013333-UpdateLaosSchoolsSchoolIndicatorsEiEFunctioningTVMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const OVERLAY_ID = 'Laos_Schools_Functioning_TV_Satellite';
+
+const OLD_NAME = 'Functioning TV, satellite receiver and dish set';
+const NEW_NAME = 'Functioning TV for teaching and learning purposes';
+
+const OLD_DATA_ELEMENT_CODE = 'SchCVD012';
+const NEW_DATA_ELEMENT_CODE = 'value';
+
+const OLD_MEASURE_BUILDER = 'valueForOrgGroup';
+const NEW_MEASURE_BUILDER = 'checkMultiConditionsByOrgUnit';
+
+const OLD_MEASURE_BUILDER_CONFIG = {
+  entityAggregation: {
+    dataSourceEntityType: 'school',
+  },
+};
+const NEW_MEASURE_BUILDER_CONFIG = {
+  entityAggregation: {
+    dataSourceEntityType: 'school',
+  },
+  dataElementCodes: ['SchCVD012', 'SchCVD012a'],
+  orgUnitConditions: {
+    Yes: {
+      SchCVD012: {
+        value: 'Yes',
+        operator: '=',
+      },
```

Comment:
I believe this will also work?

```suggestion
      SchCVD012: 'Yes'
```

similarly for the others